library/src/helpers/getExamplesFromSpec.ts

```diff
@@ -0,0 +1,14 @@
+import { Example } from '../types';
+
+export type Type = 'payload' | 'headers';
+
+export const getExamplesFromSpec = (examples: Example[], type: Type) => {
```

Comment:
You can avoid unnecessary forEach and ifs and do something like:

```ts
export const getExamplesFromSpec = (examples: Example[] = [], type: Type) => {
  return examples.map(el => el[type]).filter(Boolean);
}
```

But your solution is also good, my code is only suggestion.

Reply:
fuck man, I love it, `filter(Boolean)` is neat!

Reply:
You're welcome :)